be1-go/hub/standard_hub/message_handling.go

```diff
@@ -494,6 +494,32 @@ func (h *Hub) handleGetMessagesById(socket socket.Socket,
 	return missingMessages, getMessagesById.ID, nil
 }
 
+func (h *Hub) handleGreetServer(socket socket.Socket, byteMessage []byte) error {
+	var greetServer method.GreetServer
+
+	err := json.Unmarshal(byteMessage, &greetServer)
+	if err != nil {
+		return xerrors.Errorf("failed to unmarshal greetServer message: %v", err)
+	}
+
+	h.Lock()
+	// store information about the server
+	h.peersInfo[socket.ID()] = greetServer.Params
```

Comment:
Could be good to add methods for hub, like "AddPeersInfo", to avoid locking and unlocking every time.